### ISSUE TITLE
golangci-lint run missing print-issued-lines (gopls/vscode)

### ISSUE THREAD

**Issue by elszon** [golangci/golangci-lint#5616]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/usage/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

I don't know

### Description of the problem

The `golangci-lint run` lacks the `print-issued-lines` option used by the `gopls` to find issues. The `gopls` is also used by the VS Code.

### Version of golangci-lint

<details>

The `gopls` debug output from the VS Code:

```
2025-03-25 08:00:51.541 [error] Error while running tool: ~/go/bin/golangci-lint run --print-issued-lines=false --out-format=colored-line-number --issues-exit-code=0
2025-03-25 08:00:51.541 [error] Error: unknown flag: --print-issued-lines
Failed executing command with error: unknown flag: --print-issued-lines
```

```console
$ golangci-lint --version
golangci-lint has version v2.0.1 built with go1.24.1 from (unknown, modified: ?, mod sum: "h1:fNxppSstN0Rl5eDNRqMqdoH+/c2cr02zM8gEbhU12XQ=") on (unknown)
```

</details>

### Configuration

<details>

```console
# paste configuration file or CLI flags here
```

</details>

### Go environment

<details>

```console
$ go version && go env
go version go1.24.1 darwin/arm64
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```

</details>

### Validation

- [x] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by ldez** [comment]
Hello,

The problem will be handled by the extension: https://github.com/golang/vscode-go/issues/3732

**Comment by elszon** [comment]
Thanks.
